pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = [
     "pandas",
     "huggingface-hub",
     "gradio>=5.34.2",
+    "backoff>=2.2.1",
```

Comment:
Instead of adding a new dependency, could we just vendor in the code? my understanding is that we are just using this for this generator:

```py
def fibo():
    """Generator for Fibonacci backoff: 1, 1, 2, 3, 5, 8, ...
    """
    a, b = 1, 1
    while True:
        yield a
        a, b = b, a + b
```